dev/io.openliberty.microprofile.health.internal.common/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -17,7 +17,8 @@
                    localization="OSGI-INF/l10n/metatype">
 
     <OCD id="io.openliberty.microprofile.health" name="%mpHealth" description="%mpHealth.desc" ibm:alias="mpHealth" ibm:beta="true">
-        <AD id="fileUpdateInterval" name="%fileUpdateInterval" description="%fileUpdateInterval.desc" required="false" type="String" ibm:beta="true"/>
+        <AD id="checkInterval" name="%checkInterval" description="%checkInterval.desc" required="false" type="String" ibm:beta="true"/>
+        <AD id="startupCheckInterval" name="%startupCheckInterval" description="%startupCheckInterval.desc" required="false" type="String" ibm:beta="true"/>
```

Comment:
Good to set the following here: `ibm:type="duration(ms)" default="100ms"`

Reply:
Providing a default values prevents us from knowing if the configuration was set from the user (or provided as a default). We need user interaction to better gauge the intent on what the user wanted. With regards to the type, see above comment.